src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1304,6 +1304,54 @@ public bool IsLeftSideOfAnyAssignment(SyntaxNode node)
             return (node as ExpressionSyntax).IsLeftSideOfAnyAssignExpression();
         }
 
+        public bool IsAssignedTo(SyntaxNode node)
```

Comment:
hrmm... we already have IsWrittenTo as an extension.  Do we need this?

Note: IsWrittenTo also handles hte cases where a VB property is written to if passed to a byref/out arg..